packages/cli/src/commands/preview/server/setup.ts

```diff
@@ -200,8 +199,11 @@ async function buildManifest(
     format: "esm",
     jsx: "preserve",
     external: [
-      ...Object.keys(pkg.dependencies || {}),
-      ...Object.keys(pkg.peerDependencies || {}),
+      "./node_modules/*",
```

Comment:
@psugihara this is a better way to do this.  any thoughts on best way to do this programmatically?  naive solution I could just prepend "../" for however deep cwd is, but there is probably a better way.  is there something you can check in `process`? i.e. ask node/npm, etc?

Reply:
this seems nice and simple

I think use `process.cwd().split("/").length` and just generate the string array like that.

This plugin looks in the filesystem but this way seems quicker if the string array works https://github.com/pradel/esbuild-node-externals/blob/main/esbuild-node-externals/src/utils.ts#L23